crates/uv-resolver/src/resolver/batch_prefetch.rs

```diff
@@ -58,7 +58,6 @@ impl BatchPrefetcher {
             extra: None,
             dev: None,
             marker: None,
-            url: None,
```

Comment:
Does this file need to be changed too, to avoid batch pre-fetching for URL packages?

Reply:
Oh I see, you enforce this _outside_ of `prefetch_batches`, at the call site.